_data/collection-index.yml

```
@@ -423,3 +423,8 @@
   contact: https://github.com/vvv-interactive/ibdev/issues
   repository: https://github.com/vvv-interactive/ibdev
   ociReference: ghcr.io/vvv-interactive/ibdev
+- name: Features by Nikita Kurpas
+  maintainer: Nikita Kurpas
+  contact: https://github.com/NikitaKurpas/features/issues
+  repository: https://github.com/NikitaKurpas/features
+  ociReference: ghcr.io/NikitaKurpas/features
```

```suggestion
  ociReference: ghcr.io/nikitakurpas/features
```

Hi 👋

The ORAS spec requires the `ociReference` to be in lower case. Can we update here, thanks?
The supporting tools do help convert the case, however, its better to sync as much as possible.

---

Hey, thanks for the suggestion 👍🏻 Applied.